buildPublishable.sh

```diff
@@ -16,20 +18,20 @@ echo "Searching for stable branches with repo path refs/${repoPath}/stable/"
 origBranch=$(git symbolic-ref --short HEAD)
 
 echo "Checking if the current branch ${origBranch} is clean"
-if [ ! -z "$(git status --untracked-files=no --porcelain)" ]; then 
+if [ -n "$(git status --untracked-files=no --porcelain)" ]; then
   # Uncommitted changes in tracked files
   echo "  ERROR: Some files that have been changed are not committed!"
   echo "  This script will fail to generate a publishable build until everything has been committed"
   git status
-  exit -1
+  exit 1
 fi
 
 # initialize the destination folder
-mkdir -p ${destDir}/versions
+mkdir -p "${destDir}/versions"
 
 # Build the stable version list
-unset versionList
-for branch in `git for-each-ref --format='%(refname:short)' --sort=-refname refs/${repoPath}/stable/`; do
+versionList=""
```

Comment:
with `set -u`, `unset versionList` would have caused a failure in the `[ -z "$versionList" ]` expansion.  I suppose I could have used `[ -z ${versionList:-} ]`, but I didn't think of it at first - happy to change it if that's more legible.